packages/ember-glimmer/lib/index.js

```diff
@@ -286,3 +286,4 @@ export {
 export { setupEngineRegistry, setupApplicationRegistry } from './setup-registry';
 export { DOMChanges, NodeDOMTreeConstruction, DOMTreeConstruction } from './dom';
 export { registerMacros as _registerMacros, experimentalMacros as _experimentalMacros } from './syntax';
+export { default as ExperimentalComponentManager } from './syntax/experimental-component-manager';
```

Comment:
Not sure we need to export this, I don't think that we want folks extending from our curly component manager. We want to allow them to create their own (which is basically "just" implement XYZ hooks and go to town).